kibana_prometheus_exporter/kibana_collector.py

```diff
@@ -150,17 +150,24 @@ def _process(self) -> Iterator[Metric]:
 
 
 class KibanaCollector(object):
-    def __init__(self, host: str, path: str = "/api/status", kibana_login: str = None, kibana_password: str = None):
+    def __init__(self, host: str, path: str = "/api/status", kibana_login: str = None,
+            kibana_password: str = None, cert_path: str = None):
         self._url = url_join(host, path)
         self._kibana_login = kibana_login
         self._kibana_password = kibana_password
+        self._cert_path = cert_path
 
     def _fetch_stats(self) -> dict:
         if self._kibana_login:
             auth = (self._kibana_login, self._kibana_password)
         else:
             auth = None
-        r = get(self._url, auth=auth)
+
+        if self._cert_path:
+            r = get(self._url, auth=auth, cert=(self._cert_path))
```

Comment:
TODO(@ahmgeek): I don't think `path` is enough - maybe we need the key- , I also want to write tests for this